README.md

```diff
@@ -78,8 +79,17 @@ The Linux build uses the meson build system, and builds against system libraries
 
 6. `ninja`
 
-If you want to change the buildtype aftwerwards, you can use `meson configure --buildtype {release or debug}` in the build directory or create a secondary build directory as in Step 4
+If you want to change the buildtype afterwards, you can use `meson configure --buildtype {release or debug}` in the build directory or create a secondary build directory as in Step 4. There are also additional build options documented in the [wiki](https://github.com/cortex-command-community/Cortex-Command-Community-Project-Source/wiki/Meson-build-options) as well as through running `meson configure` in the build directory.
+
+## Installing:
+
+Follow steps 1-3 in the [building](#building) section.  
+
+4. `meson -Dno_install=False -Dinstall_fmod=True build`
```

Comment:
I assume there should be spaces here between `-D` and the stuff after it, and in the uninstall instructions below.

Reply:
No, this is correct

Reply:
Haha okay. Looks weird to not have some whitespace to me, but whatever. If it's fine it's fine :)